src/utilities/utilities.jl

```diff
@@ -421,7 +421,31 @@ const julia_remote = Remotes.GitHub("JuliaLang", "julia")
 """
 Stores the memoized results of [`getremote`](@ref).
 """
-const GIT_REMOTE_CACHE = Dict{String,Union{Remotes.Remote,Nothing}}()
+const GIT_REMOTENAME_CACHE = Dict{String,Union{String,Nothing}}()
```

Comment:
I don't think we expect `nothing` values here?

```suggestion
const GIT_REMOTENAME_CACHE = Dict{String,String}()
```